ofproto/ofproto-dpif-xlate.c

```diff
@@ -2991,6 +3000,62 @@ is_ip_local_multicast(const struct flow *flow, struct flow_wildcards *wc)
     }
 }
 
+#if defined(P4OVS)
+static int32_t
+get_fdb_data(struct xport *port, struct eth_addr mac_addr,
+             struct mac_learning_info *fdb_info)
+{
+    if (!port || !port->netdev || !port->xbundle) {
+        return -1;
+    }
+
+    memcpy(fdb_info->mac_addr, mac_addr.ea, sizeof(fdb_info->mac_addr));
+    if (port->is_tunnel) {
+        fdb_info->is_tunnel = port->is_tunnel;
+        const struct netdev_tunnel_config *underlay_tnl = NULL;
+        underlay_tnl = netdev_get_tunnel_config(port->netdev);
+        if (!underlay_tnl) {
+            VLOG_ERR("Error retrieving netdev tunnel config");
+            return -1;
+        }
+
+    int underlay_ifindex = netdev_get_ifindex(port->netdev);
```

Comment:
This line should be indented further. It's part of the `if (port->is_tunnel)` block.